server/main.js

```diff
@@ -26,6 +26,22 @@ Meteor.startup(() => {
       Azure_containerName: '',
       storageType: 'None',
     })
+  } else {
+    const settings = Settings.findOne()
+    let directive = Slingshot.getDirective(settings.storageType)._directive
```

Comment:
I get TypeError: Cannot read property '_directive' of undefined

Reply:
Were they uploaded before the initial change to profile images? If so they might be saved in the database as `https://<bucket>.s3-ca-central-1.amazonaws.com/{. . .}` rather than `https://<bucket>.s3-ca-central-1.amazonaws.com/{. . .}/image` or `https://<bucket>.s3-ca-central-1.amazonaws.com/{. . .}/thumbnail`. If so I will make a method for updating the image urls

Reply:
Yes, they were uploaded before any of the changes. I would have thought that we just saved a URL for the image and the thumbnail, like we do for images in the questions. Are we dynamically building the link to the profile image each time?